### ISSUE TITLE
Modify squashNulls doesn't work

### ISSUE THREAD

**Issue by gadesuk** [bazaarvoice/jolt#765]
Hi, 

I am trying to use the modify function squashNulls but it doesn't seem to work. It didn't work in my project, so I tried the Demo. 

Here is the input:
{
  "listPhones" : [ "01597822403", null, "01612233446" ]
}

And the spec:
[
  {
    "operation": "modify-overwrite-beta",
    "spec": {
      "listPhones": "=squashNulls"
    }
  }
]

Am I missing something or making a silly mistake?

Thanks very much

**Comment by gadesuk** [comment]
Yes, I know. I tried the test case from the code in the demo and it didn´t work. It is for that I opened this issue.

Any ideas?
Thanks

**Comment by zeshuai007** [comment]
This [PR#921](https://github.com/bazaarvoice/jolt/pull/921) fixes the problem.
Hi! @gadesuk @jsurls @milosimpson  If you have free time, welcome to review.
